Compose/Traversable

```diff
@@ -0,0 +1,24 @@
+    let Traversable = ./../Traversable/Type
+
+in  let Applicative = ./../Applicative/Type
+
+in    λ(f : Type → Type)
+    → λ(g : Type → Type)
+    → λ(fTraversable : Traversable f)
+    → λ(gTraversable : Traversable g)
+    →     ./Functor f g fTraversable.{ map } gTraversable.{ map }
+        ∧ ./Foldable f g fTraversable.{ fold } gTraversable.{ fold }
+        ∧ { traverse =
+                λ(fA : Type → Type)
+              → λ(applicative : Applicative fA)
+              → λ(a : Type)
+              → λ(b : Type)
+              → λ(k : a → fA b)
+              → fTraversable.traverse
+                fA
+                applicative
+                (g a)
+                (g b)
+                (gTraversable.traverse fA applicative a b k)
+          }
+      : Traversable (./Type f g)
```

Comment:
So, shouldn’t we be able to implement this with `../Traversable/fromTraverse` instead of `∧`? Then the Functor and Foldable instances can be implemented with just `../Traversable/extractFunctor` and `../Traversable/extractFoldable`.

Reply:
Actually, sorry – I was wrong about this. Since it uses instances on nested types, the Functor and Foldable instances are now overconstrained. Your old impl was better.

I think I also prefer `∧` in that case, because then we have only one implementation for `map` and `fold`, rather than one written directly with lesser constraints and one generated by `fromTraverse`.